plugins/modules/katello_activation_key.py

```diff
@@ -159,7 +179,7 @@ def main():
             )),
             content_overrides=dict(type='list', elements='dict', options=dict(
                 label=dict(required=True),
-                override=dict(choises=['enabled', 'disabled']),
```

Comment:
How can this ever have worked...
(I know it just skipped the choices validation)

Reply:
The joys of `**kwargs`